orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/ExpressionAware.kt

```diff
@@ -62,6 +63,21 @@ interface ExpressionAware {
     return this
   }
 
+  /**
```

Comment:
The idea here is that once `parameters` have been processed, they are inserted into `trigger`.
This helps route the call to the right place using the `RuntimeExpressionUpdater` pattern.